### ISSUE TITLE
关于 training.py 文件

### ISSUE THREAD

**Issue by keep-eyes-clean** [yaochenzhu/LLM4Rec#7]
for epoch in range(num_pretrained_epochs):
      review_total_loss = 0
      
      # Initialize tqdm progress bar
      # progress_bar = tqdm(review_data_loader, desc=f"Epoch {epoch + 1}", 
      #                     disable=not accelerator.is_local_main_process, ncols=80)
      for input_ids_prompt, input_ids_main, attention_mask in review_data_loader:
          review_optimizer.zero_grad()

          # Obtain the data
          input_ids_prompt = input_ids_prompt.to(device)
          input_ids_main = input_ids_main.to(device)
          attention_mask = attention_mask.to(device)

          # Forward pass
          outputs = content_model(input_ids_prompt, 
                                  input_ids_main, 
                                  labels_main=input_ids_main,
                                  attention_mask=attention_mask)
          review_loss = outputs[0]

          # Backward pass and optimization
          accelerator.backward(review_loss)
          review_optimizer.step()

          review_total_loss += review_loss.item()
          # progress_bar.set_postfix({"Review Loss": review_loss.item()})

我尝试把这个代码独立出来训练，我将学习率设定为1e-4，batch设定为20， 为什么我的训练中发现的loss/batch是一直在波动的，模型训练不稳定，能请教一下该怎么做才能解决这个不稳定的问题呢？
